tests/unit/time_series_test.coffee

```diff
@@ -71,3 +72,168 @@ test 'Margins are the right size when showLegend is no', (assert) ->
   assert.equal @subject().get('marginRight'), 0, 'no right margin'
   assert.equal @subject().get('marginBottom'), 0,
     'no bottom margin if showLegend is no'
+
+test 'The number of labels should be equal to the number of day intervals',
+(assert) ->
+  timeSeriesContentLine =
+    yAxisFromZero: no
+    dynamicXAxis: no
+    selectedInterval: 'D'
+    lineData: [
+        time: d3.time.format('%Y-%m-%d').parse "2011-04-1"
+        value: 3190223.5507746544
```

Comment:
these values look ugly and there's no real reason for that right? why not make them like 1 or 0 or something that isn't 20 digits long ^^